### ISSUE TITLE
Use V2 provider with Infura relay url failed

### ISSUE THREAD

**Issue by JustinFeng** [WalletConnect/walletconnect-monorepo#1720]
**Describe the bug**

We are trying to continue using Infura WebSocket server as for the relay URL (because walletConnect default relay is blocked in our country) after upgrade to WalletConnect V2 provider

- wss://mainnet.infura.io/ws/v3/KEY

but the WebSocket connection cannot be maintained, we are seeing the following error in console

<img width="910" alt="image" src="https://user-images.githubusercontent.com/1527902/205867271-444b4fff-c85d-4d3e-9b57-4c9b01339e7d.png">

also had a close look into the WebSocket interaction, looks like the v2 provider is sending the following message to the server

```
{"id":1670319570538786,"jsonrpc":"2.0","method":"irn_subscribe","params":{"topic":"17099eee901dcc115fd7f33aa6747e7c6b883eed314b7e35a8976cd985065ff4"}}
```

but seems Infura doesn't know how to handle this and ends up closing the connection, we also experimented with QuickNode server, and it behaves the same.

Wondering does this mean the v2 provider can only use the WalletConnect relay server as of now, or if there is any more detailed document to explain how to use the v2 provider with Infura.

**SDK Version (if relevant)**
 - Client: JS

**To Reproduce**
you should be able to see the error in the console once you create the provider with Infura relay URL

```
const provider = await UniversalProvider.init({
      logger: "info",
      projectId,
      relayUrl: "wss://mainnet.infura.io/ws/v3/KEY",
    });
```

**Expected behavior**
We expect the relay WebSocket can be connected properly so that we can use the provider.

**Desktop (please complete the following information):**
 - OS: MacOS 13.0.1 (22A400)
 - Browser Chrome Version 108.0.5359.94 (Official Build) (arm64)

**Comment by ganchoradkov** [comment]
hey @JustinFeng thanks for the report

`relayUrl` is the URL of the actual relay server that manages communication between 2 clients
`Infura` is a node provider that providers blockchain data which is completely different.

For your case please try setting `relayUrl` to `ws://relay.walletconnect.org`

and regarding infura, you can configure the chain providers in the namespace arguments of `.connect()`

example:
```
const provider = await UniversalProvider.init({
      logger: "info",
      projectId,
      relayUrl: "ws://relay.walletconnect.org",
    });

 //  create sub providers for each namespace/chain
await provider.connect({
  namespaces: {
    eip155: {
      methods: [
        "eth_sendTransaction",
        "eth_signTransaction",
        "eth_sign",
        "personal_sign",
        "eth_signTypedData",
      ],
      chains: ["eip155:1"],
      events: ["chainChanged", "accountsChanged"],
      rpcMap: {
        1: "mainnet.infura.io/ws/v3/KEY", // <--- infura url comes here
      },
    },
  },
});
```

**Comment by JustinFeng** [comment]
Hi @ganchoradkov, thanks for explaining that Infura cannot do the relay server, just in our case as I mentioned `relay.walletconnect.org` is blocked in our country (China), wondering if there is any other relay server that the v2 provider can connect to, or is it possible to not use a relay in v2 provider? cheers

**Comment by chadyj** [comment]
@JustinFeng Just to clarify:
We believe `relay.walletconnect.com` is blocked. (note - .COM)
But you can use `relay.walletconnect.org` instead. (note - .ORG)

Can you please confirm if `.ORG` works for you?

**Comment by JustinFeng** [comment]
Hi @chadyj , thanks for the reply, my bad that I didn't notice you guys are suggesting `.org`. 

I had some testing, `.org` relay is connectable from China so that I can see the QRcode modal pop up, but when I scan it with my wallet (TrustWallet) on my phone, it hangs. I also tested by putting my phone on a VPN and then scanning the QRcode, which turns out to work. My gut feeling is there is still something blocked on the wallet side, any thoughts on this?

(btw I verified WalletConnect v1 works fine without VPN on the phone)
